docs/user_guide/1_1_pelicase.rst

```diff
@@ -0,0 +1,53 @@
++-----------------------------------------------------------------------+--------------------------------------+
+| Mounting Plate screws for the hand                                    |     8                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| 5m Ethernet cable                                                     |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| "NUC-CONTROL" USB-ethernet adapter                                    |     1   (only 1 per bimanual system) |
++-----------------------------------------------------------------------+--------------------------------------+
+| "SERVER" USB-ethernet adapter                                         |     1   (only 1 per bimanual system) |
++-----------------------------------------------------------------------+--------------------------------------+
+| "RIGHT/LEFT HAND" Labelled USB-ethernet adapter                       |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| NUC labelled SHADOW NUC-CONTROL for running hand’s driver and the arm |     1   (only 1 per bimanual system) |
++-----------------------------------------------------------------------+--------------------------------------+
+| NUC PSU                                                               |     1   (only 1 per bimanual system) |
++-----------------------------------------------------------------------+--------------------------------------+
+| NUC PSU power lead according to destination country                   |     1   (only 1 per bimanual system) |
++-----------------------------------------------------------------------+--------------------------------------+
+| Toolbox (Contains hex drivers to robot maintenance)                   |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Cut allen key (inside the Toolbox)                                    |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Allen key (inside the Toolbox)                                        |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Spooltool (inside the Toolbox)                                        |     1                                |
++-----------------------------------------------------------------------+--------------------------------------+
+| Luggage locks for pelicase                                            |     2                                |
```

Comment:
In the tile `pelicase` is spelled using 2 words. Which one is the correct one?

Reply:
and "Contains" with lower case maybe?

Reply:
both done